src/common/OneDSLoggerTelemetry/oneDSLogger.ts

```diff
@@ -174,37 +181,37 @@ export class OneDSLogger implements ITelemetryLogger {
                     case 'za':
                     case 'ae':
                     case 'kr':
-                        instrumentationSettings.endpointURL = 'https://us-mobile.events.data.microsoft.com/OneCollector/1.0/',
-                            instrumentationSettings.instrumentationKey = '197418c5cb8c4426b201f9db2e87b914-87887378-2790-49b0-9295-51f43b6204b1-7172'
+                        instrumentationSettings.endpointURL = 'https://us-mobile.events.data.microsoft.com/OneCollector/1.0/';
```

Comment:
good to store as constant in the common file

Reply:
Done